pgbouncer/build-osx.sh

```diff
@@ -90,72 +90,51 @@ cat<<PGBOUNCER > $WD/pgbouncer/build-pgbouncer.sh
 
     cd $PG_PATH_OSX/pgbouncer/source/pgbouncer.osx/
     
-    CFLAGS="$PG_ARCH_OSX_CFLAGS -arch i386 -O2" LDFLAGS="-arch i386" MACOSX_DEPLOYMENT_TARGET=10.6 ./configure --prefix=$PG_PATH_OSX/pgbouncer/staging/osx.build/pgbouncer --with-libevent=/opt/local/Current --with-openssl=/opt/local/Current || _die "Failed to configure pgbouncer"
-    mv lib/usual/config.h lib/usual/config_i386.h || _die "Failed to rename config.h"
-
-    CFLAGS="$PG_ARCH_OSX_CFLAGS -arch x86_64 -O2" LDFLAGS="-arch x86_64" MACOSX_DEPLOYMENT_TARGET=10.6 ./configure --prefix=$PG_PATH_OSX/pgbouncer/staging/osx.build/pgbouncer --with-libevent=/opt/local/Current || _die "Failed to configure pgbouncer"
+    CFLAGS="$PG_ARCH_OSX_CFLAGS -arch x86_64 -O2" LDFLAGS="-arch x86_64" MACOSX_DEPLOYMENT_TARGET=$MACOSX_MIN_VERSION LIBEVENT_LIBS="-L/opt/local/Current_v15/lib -levent" LIBEVENT_CFLAGS="-I/opt/local/Current_v15/include" ./configure --prefix=$PG_PATH_OSX/pgbouncer/staging/osx.build/pgbouncer --with-openssl=/opt/local/Current_v15 || _die "Failed to configure pgbouncer"
     mv lib/usual/config.h lib/usual/config_x86_64.h || _die "Failed to rename config.h"
 
-    CFLAGS="$PG_ARCH_OSX_CFLAGS -arch i386 -arch x86_64 -O2" LDFLAGS="-arch i386 -arch x86_64" MACOSX_DEPLOYMENT_TARGET=10.6 ./configure --prefix=$PG_PATH_OSX/pgbouncer/staging/osx.build/pgbouncer --with-libevent=/opt/local/Current || _die "Failed to configure pgbouncer"
+    CFLAGS="$PG_ARCH_OSX_CFLAGS -arch arm64 -O2" LDFLAGS="-arch arm64" MACOSX_DEPLOYMENT_TARGET=$MACOSX_MIN_VERSION LIBEVENT_LIBS="-L/opt/local/Current_v15/lib -levent" LIBEVENT_CFLAGS="-I/opt/local/Current_v15/include" ./configure --prefix=$PG_PATH_OSX/pgbouncer/staging/osx.build/pgbouncer --with-openssl=/opt/local/Current_v15 || _die "Failed to configure pgbouncer"
+    mv lib/usual/config.h lib/usual/config_arm64.h || _die "Failed to rename config.h"
+
+    CFLAGS="$PG_ARCH_OSX_CFLAGS -arch x86_64 -arch arm64 -O2" LDFLAGS="-arch x86_64 -arch arm64" MACOSX_DEPLOYMENT_TARGET=$MACOSX_MIN_VERSION LIBEVENT_LIBS="-L/opt/local/Current_v15/lib -levent" LIBEVENT_CFLAGS="-I/opt/local/Current_v15/include" ./configure --prefix=$PG_PATH_OSX/pgbouncer/staging/osx.build/pgbouncer --with-openssl=/opt/local/Current_v15 || _die "Failed to configure pgbouncer"
```

Comment:
again $PG_ARCH_OSX_CFLAGS already contains both archs, so do not need to mention "-arch x86_64 -arch arm64 -O2" explicitly again